apps/src/code-studio/components/progress/ProgressDot.jsx

```diff
@@ -202,8 +215,8 @@ export const ProgressDot = Radium(React.createClass({
 
     // fa-list-ol is used only by our redesigned dots, but we don't want to use
     // it here
```

Comment:
The `fa-list-ol` special case comment here should maybe be moved up to your mapping.